Makefile

```diff
@@ -92,6 +94,21 @@ TRIVY_VERSION ?= 0.14.0
 PROTOC_VERSION ?= 3.15.2
 SHELLCHECK_VER ?= v0.7.2
 
+
+# For aws integration tests 
+BUILD_TIMESTAMP_W_SEC := $(shell date +%Y-%m-%d-%H-%M-%S)
+EKS_CLUSTER_NAME := integ-cluster-$(BUILD_TIMESTAMP_W_SEC)
```

Comment:
As this is only required for cluster create, could we move these variables to the script instead of keeping in the `Makefile`?